framework/test/unit/components/storage/entities/round.js

```diff
@@ -117,9 +117,9 @@ describe('Round', () => {
 		SQLs = RoundEntity.SQLs;
 	});
 
-	beforeEach(() => seeder.seed(storage));
+	beforeEach(async () => seeder.seed(storage));
 
-	afterEach(() => {
+	afterEach(async () => {
```

Comment:
If that's the problem then we are doing same on many other places as well. But personally I don't feel this is the issue. Mocha treat a function which return Promise as builtin feature.

Reply:
@nazarhussain You are right here. I'll revert these changes.